hacking/build_library/build_ansible/command_plugins/dump_keywords.py

```diff
@@ -0,0 +1,121 @@
+# coding: utf-8
+# Copyright: (c) 2019, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import importlib
+import os.path
+import pathlib
+import re
+from distutils.version import LooseVersion
+
+import jinja2
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+from ansible.module_utils._text import to_bytes
+
+# Pylint doesn't understand Python3 namespace modules.
+from ..change_detection import update_file_if_different  # pylint: disable=relative-beyond-top-level
+from ..commands import Command  # pylint: disable=relative-beyond-top-level
+
+
+DEFAULT_TEMPLATE_DIR = pathlib.Path(__file__).absolute().parents[4] / 'docs/templates'
+TEMPLATE_FILE = 'playbooks_keywords.rst.j2'
+PLAYBOOK_CLASS_NAMES = ['Play', 'Role', 'Block', 'Task']
+
+
+def load_definitions(keyword_definitions_file):
+    docs = {}
+    if keyword_definitions_file:
```

Comment:
Why not do
```suggestion
    if not keyword_definitions_file:
        return {}
```

This'd reduce the nesting of the following block.

Reply:
Looking at the code, it seems like the condition should be moved to the caller.  If keyword_defintions_file was optional here, it would make sense to keep the condition in load_definitions().  keyword_definitions_file feels like it should remain mandatory because this function can't do anything if the file isn't specified.

Reply:
Done